cppForSwig/TxClasses.cpp

```diff
@@ -432,10 +456,10 @@ void Tx::unserializeWithRBFFlag(const BinaryData& rawTx)
 /////////////////////////////////////////////////////////////////////////////
 BinaryData Tx::getThisHash(void) const
 {
-   if (thisHash_.getSize() == 32)
-      return thisHash_;
+	if (thisHash_.getSize() == 32)
```

Comment:
try to avoid white space changes in the future. set your GDE to tabs as spaces and 3 spaces per tabs